### PR TITLE
Add blogpost and report for Kyvernos fuzzing audit

### PR DESCRIPTION
**Pull request by AdamKorcz**
## Proposed Changes
Adds the blogpost and the report for Kyvernos 2023 fuzing security audit.

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

**Review by realshuting — APPROVED** [review]
Thanks @AdamKorcz !

File: content/en/blog/general/fuzzing-audit/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
...rote a fuzzer for each routine which passes both a valid Kyverno Cluster Policy as well pseduo-random unstructured Kubernetes resource, testing these validation routines for both reliability and security issues. One of the issues that the fuzzers found during the audit was triggered by Kyvernos mishandling of the input resource. The Validate, Mutate and Verify Image fuzzers require a substantial setup depending on multiple mocked types, and Ada Logics did several iterations to improve their effectiveness. This included several code optimizations in the fuzzers including how the fuzzers generate policy rules and the Kubernetes resources before passing them onto the entrypoints. Another optimization was to add a dictionary; Native Go fuzzing does not support dictionaries, but OSS-Fuzz does when running the fuzzers. A dictionary is a list of tokens that the fuzzing engine will mutate over in the first iterations when starting to run a fuzzer, and it can be a great resource to get a fuzzer started with meaningful test cases. In Kyvernos case, Ada Logics scraped the json tags of several Kubernetes resource types so that the fuzzers can use these when generating the Kubernetes resources.
```
Comment (chipzoller) [inline content/en/blog/general/fuzzing-audit/index.md:24]: "Kyverno's"

Comment (chipzoller) [reply content/en/blog/general/fuzzing-audit/index.md:24]: "JSON tags"

File: content/en/blog/general/fuzzing-audit/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Besides the far-reaching Validate, Mutate and Verify Image fuzzers, the auditors added fuzzers for other high-level APIs and complex targets in Kyverno. 
+
+All of Kyvernos fuzzers run continuously on [OSS-Fuzz](https://github.com/google/oss-fuzz) that dedicates vast resources to run Kyvernos fuzzer at scale. When a fuzzer finds a crash, OSS-Fuzz notifies the Kyverno maintainers with a crash report and a stack trace.
```
Comment (chipzoller) [inline content/en/blog/general/fuzzing-audit/index.md:28]: "Kyverno's"

File: content/en/blog/general/fuzzing-audit/index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+The fuzzers found 3 bugs during the audit. They were all found by OSS-Fuzz after the fuzzers started running continuously. Ada Logics fixed the three issues. The report includes notes from the triaging of the issues and links to the fixes.
```
Comment (chipzoller) [inline content/en/blog/general/fuzzing-audit/index.md:30]: "three bugs"

**Review by chipzoller — CHANGES_REQUESTED** [review]
Great post, thank you! Just a few very minor things.

File: content/en/blog/general/fuzzing-audit/index.md (hunk 1 above)
Comment (JimBugwadia) [reply content/en/blog/general/fuzzing-audit/index.md:24]: Add link to https://github.com/kyverno/kyverno/tree/main/test/fuzz?

File: content/en/blog/general/fuzzing-audit/index.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
+date: 2023-09-06
+title: "Kyverno Completes Fuzzing Security Audit"
+linkTitle: "Kyverno Completes Fuzzing Security Audit"
+author: Adam Korczynski
+description: "Presenting the results from the fuzzing security audit"
+---
+
+Kyverno is happy to announce the completion of its fuzzing security audit. The audit was carried out by [Ada Logics](https://adalogics.com/) and is part of an initiative by the CNCF to bring fuzzing to the CNCF landscape; Fuzzing is an important part in keeping CNCF projects secure and robust, and it has found security vulnerabilities and reliability issues [in several other CNCF-hosted projects](https://www.cncf.io/blog/2023/04/18/cncf-fuzzing-open-source-projects-for-security-and-reliability/). The audit spanned July and August of 2023 and resulted in 15 fuzzers written for the Kyverno project. The fuzzers found three bugs during the audit itself and OSS-Fuzz will continue to run them after the audit has concluded to test Kyverno for bugs and vulnerabilities.
```
Comment (JimBugwadia) [inline content/en/blog/general/fuzzing-audit/index.md:9]: Kyverno, a CNCF policy engine for Kubernetes, is happy...

**Review by JimBugwadia — COMMENTED** [review]
Looks good, minor comments.

**Review by chipzoller — APPROVED** [review]

**Pull request merged**